publisher/start.sh

```diff
@@ -34,7 +34,27 @@ EORESTORE
       pg_restore /dumps/db.backup -U postgres -d ${DB_NAME}
 EORESTORE
 
+    # we need to sanitize sensitive data first
+    docker run --name sanitizing_db_container -d --rm -v ${DUMPS_DIR}:/files \
+    -e POSTGRES_DB=${DB_NAME} -e POSTGRES_PASSWORD= postgres:10.5
+    # wait for "database system is ready to accept connections", hoping 4 sec is enough
+    sleep 4;
 
+    docker exec sanitizing_db_container psql -f /files/${backup_roles_file} -U postgres
+    docker exec sanitizing_db_container pg_restore /files/${backup_file} -U postgres -d ${DB_NAME}
+
+    cat <<-EOSANITIZE | (docker exec -i sanitizing_db_container sh -c "cat > sanitize.sql")
```

Comment:
suggestion: Maybe store this script as an SQL file. I foresee it will be more complex